var/spack/repos/builtin/packages/py-plotly/package.py

```diff
@@ -10,10 +10,11 @@ class PyPlotly(PythonPackage):
     """An interactive, browser-based graphing library for Python"""
 
     homepage = "https://plot.ly/python/"
-    url      = "https://github.com/plotly/plotly.py/archive/v2.2.0.tar.gz"
+    url      = "https://pypi.io/packages/source/p/plotly/plotly-3.0.0.tar.gz"
 
-    version('3.0.0', sha256='aa8e0020e139f1a3413df578ec3e3162c6ccb2d4c6d66f6fd98547c3ff83ce0b')
-    version('2.2.0', sha256='dad2a49fe355dddb6ae159e96c10ac22413a33cbac513b4dbf3791e63ec33c1f')
+    version('4.1.0', sha256='9333f28feacaa0751df04cccfb266463bf5029dcb106d3f81d7af5942fd3d998')
+    version('3.0.0', sha256='baab7f8d54f37df89f31d6e6e5d2d4b491bccb79e5d490e25d5762bb762a277c')
+    version('2.2.0', sha256='ca668911ffb4d11fed6d7fbb12236f8ecc6a7209db192326bcb64bdb41451a58')
```

Comment:
Seems like there are more dependencies here:
```
pkg_resources.DistributionNotFound: The 'retrying>=1.3.3' distribution was not found and is required by plotly
```

Reply:
Thank you! it helped. I also rebased the PR on the latest `develop`. Manually checked that everything works via `source /gpfs/bbp.cscs.ch/apps/hpc/jenkins/pulls/930/config/modules.sh`.